src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java

```diff
@@ -36,22 +36,22 @@
 
   static {
     SPECIAL_PAGE_SLOTS_SELECTORS = new HashMap<>();
-    SPECIAL_PAGE_SLOTS_SELECTORS.put(TOP_LB, "#TOP_LEADERBOARD");
+    SPECIAL_PAGE_SLOTS_SELECTORS.put(TOP_LB, "#top_leaderboard");
 
     FILE_PAGE_SLOTS_SELECTORS = new HashMap<>();
-    FILE_PAGE_SLOTS_SELECTORS.put(TOP_LB, "#TOP_LEADERBOARD");
-    FILE_PAGE_SLOTS_SELECTORS.put(TOP_BOXAD, "#TOP_BOXAD");
-    FILE_PAGE_SLOTS_SELECTORS.put(BOTTOM_LB, "#BOTTOM_LEADERBOARD");
+    FILE_PAGE_SLOTS_SELECTORS.put(TOP_LB, "#top_leaderboard");
+    FILE_PAGE_SLOTS_SELECTORS.put(TOP_BOXAD, "#top_boxad");
+    FILE_PAGE_SLOTS_SELECTORS.put(BOTTOM_LB, "#bottom_leaderboard");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "#bottom_leaderboard" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)